src/contexts/LangContext.ts

```diff
@@ -1,8 +1,5 @@
-import namedContext from './_namedContext';
+import {Lang} from '../models';
 
-export enum Lang {
```

Comment:
deleting by export is a breaking change, we need to save the export in this place and add //TODO: remove it when there is a new major